imagebitmap-renderingcontext/WEB_FEAUTRES.yml

```
@@ -0,0 +1,3 @@
+features:
+- name: createimagebitmap
```

```suggestion
- name: imagebitmaprenderingcontext
```

The `createImageBitmap()` global method is to create an `ImageBitmap` object which isn't the same thing.

---

Ah, thanks!